scripts/ci-e2e-lib.sh

```diff
@@ -18,21 +18,22 @@
 capi:buildDockerImages () {
   # Configure provider images generation;
   # please ensure the generated image name matches image names used in the E2E_CONF_FILE
+  ARCH="$(go env GOARCH)"
   export REGISTRY=gcr.io/k8s-staging-cluster-api
   export TAG=dev
-  export ARCH=amd64
+  export ARCH
```

Comment:
```suggestion
  export ARCH="$(go env GOARCH)"
```

Nit

Reply:
shell checker was complying when I originally inlined export and $(), this was the suggested solution

Reply:
The problem is that the error is masked if you have it in one line